app/DoctrineMigrations/Version20160803183748.php

```diff
@@ -31,11 +32,12 @@ public function up(Schema $schema)
     public function down(Schema $schema)
     {
         // this down() migration is auto-generated, please modify it to your needs
-        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
```

Comment:
Ah ok. Wasn't sure about this. Maybe it's because of doctrine update. Doesn't really matter.